ci/pipeline.yml

```diff
@@ -121,13 +121,15 @@ jobs:
       - task: unit
         file: repo/ci/tasks/unit.yml
 
-  - name: ship-develop
+  - name: continuously-ship-develop
```

Comment:
@mumoshu  how about 'auto-ship-develop'??  shorter name could be better for many ppl.